### DISCUSSION TITLE
Gatsby Webpack Cache Not Resolving  postcss-loader/dist/cjs

### DISCUSSION THREAD

**Discussion by LinnJS** [tailwindlabs/tailwindcss#6370] [Help]
[webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Resolving 'postcss-loader/dist/cjs'

I have been having this error since Tailwind V2 and just upgraded to V3 hoping this was something that would just go away hoping it was a dependency bug or something but its still happening and I am tired of seeing it. Does anyone know how to fix this?

Here is a [link](https://github.com/LinnJS/suncoast-masons) to the open source project

![image](https://user-images.githubusercontent.com/23741323/145601590-554797e8-55b1-4aed-9130-455adfba494c.png)

**Comment by dtcbyron** [comment]
Did you solve this? Just upgraded to gatsby v4 and seeing this issue now

**Reply by LinnJS** [reply]
@byronlanehill I have not found a resolution on this at all, not 100% sure if its a tailwind issue, postcss issue, or Gatsby issue.

**Reply by davshoward** [reply]
It if helps - I'm having the same warning without using Tailwind (using CSS modules).

**Reply by gsavchenko** [reply]
I am also having this issue when attempting to migrate from Gatsby V3 to V4 and I don't see very many solutions or documentation on what the issue is. It seems to be happening for all packages that I have not just a few that I've seen called out.

**Comment by LinnJS** [comment]
I escalated to be an issue [here](https://github.com/tailwindlabs/tailwindcss/issues/6669)
